### ISSUE TITLE
Noob-Documentation

### ISSUE THREAD

**Issue by cstenkamp** [frankier/skelshop#33]
Hello Frankie,
As I've needed to spend quite some time reading into this codebase and we don't want that to be a waste, I've discussed with Peter that it may make sense to create a "Noob-Documentation". As I've for example never even encountered Snakemake or Singularity before, I think it makes sense to explain things at a very basic level, additionally to the (very helpful, believe me, without it I'd be completely lost!) documentation you've created in the past few days. So would you be fine with me adding a few pages to your documentation?

**Comment by frankier** [comment]
Yes please! It seems from my slightly rushed work so far on the docs that they can always be improved. I did already try to divide the docs into two (user/dev) and try to make the first more approachable, but yes there could be more context, more "what is it?" and "why is it like this?" (why oh why is it like this :wink:).
